### PR TITLE
3mdeb contrib q1

### PR DESCRIPTION
**Pull request by artur-rs**

### CODE REVIEW

File: blog/content/post/2020-04-16-3mdeb-contrib-Q1.md [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+
+---
+
+## Intro
+
+Our mission @3mdeb is to develop secure and maintainable firmware and
+applications helping our clients reach their goals. During everyday work
+sometimes we struggle with frustrating problems that shouldn't occur in the
+first place. When we finally deal with the issues, the first thing we want to do
+is to share the good news with the related community. This is our main
```
Comment (pietrushnic) [inline blog/content/post/2020-04-16-3mdeb-contrib-Q1.md:29]:
```suggestion
is to share the good news with the related communities. This is our main
```

File: blog/content/post/2020-04-16-3mdeb-contrib-Q1.md [hunk 2]
```diff
@@ -0,0 +1,259 @@
...
+
+    Full list of 2020'Q1 patches:
+
+    Author          | Category | Patch | URL
+    ----------------|----------|-------|----
+    Krystian Hebel  | Build    | Move bootloader data out of measured block | [link][39]
+    Krystian Hebel  | Build    | Use more hidden symbols to fix 32bit boot | [link][40]
+    Michał Żygowski | Security | Add sha256 | [link][41]
+    Michał Żygowski | README   | Add basic readme with Travis build status | [link][42]
+    Krystian Hebel  | Main     | Move PCR extension logic to a separate function | [link][43]
```
Comment (pietrushnic) [inline blog/content/post/2020-04-16-3mdeb-contrib-Q1.md:125]: @artur-rs I though there was my patch also, but may it was merged before Q1?

File: blog/content/post/2020-04-16-3mdeb-contrib-Q1.md [hunk 3]
```diff
@@ -0,0 +1,259 @@
...
+4. [meta-virtualization](https://git.yoctoproject.org/cgit/cgit.cgi/meta-virtualization/)
+
+    > This layer enables hypervisor, virtualization tool stack, and cloud
+    > support.
+
+    Full list of 2020'Q1 patches:
+
+    Author     | Category | Patch | URL
+    -----------|----------|-------|----
+    Piotr Król | dev86    | Implement IVRS IVHD type 11h parsing | [link][45]
```
Comment (pietrushnic) [inline blog/content/post/2020-04-16-3mdeb-contrib-Q1.md:156]: @artur-rs copy paster error, patch description incorrect

File: blog/content/post/2020-04-16-3mdeb-contrib-Q1.md [hunk 4]
```diff
@@ -0,0 +1,259 @@
...
+into the details of pointed technologies. Also, if you have any questions
+regarding the basics of the contribution process or simply want to send your
+first patches to the world of the open source IT but you lack the courage to do
+so, we will be glad to help you out.
+
+## Summary
+
+Do you still hesitate to be a part of the open source community? By contribution
+you not only share valuable code but improve your software by the experienced
+reviewers, take part in the act of learning and teaching by explaining how you
```
Comment (pietrushnic) [inline blog/content/post/2020-04-16-3mdeb-contrib-Q1.md:195]:
```suggestion
you not only share valuable code but improve your software through review of
experienced community members, take part in the act of learning and teaching by explaining how you
```

File: blog/content/post/2020-04-16-3mdeb-contrib-Q1.md [hunk 5]
```diff
@@ -0,0 +1,259 @@
...
+do things in the example project and build a reputation around people who are
+interested in similar things. Do not wait and join the open source family!
```
Comment (pietrushnic) [inline blog/content/post/2020-04-16-3mdeb-contrib-Q1.md:197]: @artur-rs second sentence of this paragraph is way too long, let's split it.

File: blog/content/post/2020-04-16-3mdeb-contrib-Q1.md (hunk 2 above)
Comment (artur-rs) [reply blog/content/post/2020-04-16-3mdeb-contrib-Q1.md:125]: @pietrushnic it is marked as `7/22/2019`

File: blog/content/post/2020-04-16-3mdeb-contrib-Q1.md [hunk 6]
```diff
@@ -0,0 +1,261 @@
...
+
+    >  coreboot is an extended firmware platform that delivers a lightning fast
+    >  and secure boot experience on modern computers and embedded systems. As
+    >  an Open Source project it provides auditability and maximum control over
+    >  technology.
+
+    This project is the unquestioned number one in this overview - to sum up,
+    the most significant changes:
+
+    * active development of AMD DRTM
```
[the remark this answers is not in the document]
Comment (artur-rs) [reply blog/content/post/2020-04-16-3mdeb-contrib-Q1.md:58]: i was not 100% sure about this note, thanks - the patches misleads me

**Pull request merged**